sdk/communication/Azure.Communication.CallAutomation/tests/CallAutomationClient/CallAutomationClientLiveTests.cs

```diff
@@ -118,5 +118,59 @@ public async Task CreateCallToPSTNGetCallAndHangUpCallTest()
                 Assert.Fail($"Unexpected error: {ex}");
             }
         }
+
+        [Test]
+        public async Task CreateCallWithMediaStreaming()
+        {
+            //if (SkipCallingServerInteractionLiveTests)
```

Comment:
Any reason that this is commented out but not removed? Also why this is not needed?

Reply:
I reinstated it, was a miss from my part :)